packages/dashboard-frontend/src/services/bootstrap/issuesReporter.ts

```diff
@@ -12,12 +12,15 @@
 
 import { injectable } from 'inversify';
 
-export type IssueType = 'cert' | 'sso' | 'unknown';
-export type Issue = {
+export type IssueType = 'cert' | 'sso' | 'workspaceStopped' | 'unknown';
+export type Issue<T = unknown> = {
   type: IssueType;
   error: Error;
+  data?: T;
```

Comment:
Hello @olexii4 @akurinnoy , how does adding `data` here this look? I added this to pass extra links for when rendering the error page:
https://github.com/eclipse-che/che-dashboard/blob/d685db23d4cfb2d1548e9290f1c3126ab75beb8b/packages/dashboard-frontend/src/Layout/ErrorReporter/Issue/index.tsx#L35-L39

Reply:
I don't think that we are going to extend this part very often
As for me, I prefer to use just `WorkspaceRoutes` until we will need something else. I don't think we will extend this part in the near future.

@akurinnoy WDYT?

Reply:
> As for me, I prefer to use just WorkspaceRoutes until we will need something else.

Do you mean add a private field for `WorkspaceRoutes` inside of `IssuesReporterService`?

https://github.com/eclipse-che/che-dashboard/blob/4a4d43c388e6638aea04705931472aa12e21c10c/packages/dashboard-frontend/src/services/bootstrap/issuesReporter.ts#L25

Reply:
Thank you for the suggestion, in my latest push, I've changed:
```
data?: T;
```
to
```
data?: WorkspaceRoutes
```